app/trainingScheduler/mytrainings/mytrainings.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input, OnInit} from "@angular/core";
+import { RouterModule, Router } from "@angular/router";
+
+import { TrainingsMainService } from "../trainings/trainingsmain.service";
+import { TrainingListingComponent } from "../sharedcomponent/trainingListComp.component";
+import { LoginComponent } from "../../login/login.component";
+
+@Component({
+  selector: "my-trainings",
```

Comment:
This component is for listing the user's trainings and so component name is given as "my-trainings"